api/src/main/java/org/eclipse/microprofile/config/Config.java

```diff
@@ -104,6 +110,8 @@
      * Return the resolved property value with the specified type for the
      * specified property name from the underlying {@link ConfigSource ConfigSources}.
      *     
+     * Registered converters are applied if they match the propertyType class
```

Comment:
What do you mean by saying 'Registered converters'? It depends. If the registered converter is having the lower priority than the implicit converter, it will not be used.